lib/inspec/formatters/base.rb

```diff
@@ -155,7 +155,7 @@ def format_example(example) # rubocop:disable Metrics/AbcSize
         code_desc: code_description,
         run_time: example.execution_result.run_time,
         start_time: example.execution_result.started_at.to_datetime.rfc3339.to_s,
-        resource_title: example.metadata[:described_class].to_s || example.metadata[:example_group][:description].to_s,
+        resource_title: example.metadata[:example_group][:description],
```

Comment:
I think we still need this fallback as I remember adding this in because some instances of `example.metadata[:example_group][:description]` were empty.

Reply:
Hmm this would require a lot of code changes as the class is referred in `cli` reporter for the headings. Doing a simple `to_s` would not fix that. I did not know that this was a class when I did my initial commit. This might take some time to get going. I might de-prioritize this as we have the YAML reporter now. Thanks for all the help.